### ISSUE TITLE
crash if header is in list items

### ISSUE THREAD

**Issue by clkao** [arve0/markdown-it-header-sections#1]
it seems this crashes markdown-it-header-sections:

``` markdown
- foo
  ### sss

- bar
  ### bar
```

**Comment by arve0** [comment]
Thanks for the heads up. I don't have any time to look into this today, but I will accept a PR if you are interested in hunting the bug down.

**Comment by clkao** [comment]
#2 now has the failing test case.  won't be able to investigate further for now.

**Comment by arve0** [comment]
Thanks, I'll look into it.
22. sep. 2015 06:23 skrev "Chia-liang Kao" notifications@github.com:

**Comment by arve0** [comment]
One way to fix this issue is by recording the nested state when opening sections. Something like:

```
openingState = {
  li: 1,
  ... }
```

Which should allow for headers inside other elements.

**Comment by arve0** [comment]
Fixed in v0.2.0
